app/build.gradle

```
@@ -30,6 +30,7 @@ android {
         debug {
             debuggable true
             minifyEnabled false
+            applicationIdSuffix = ".debug"
```

sorry for being late, but can we please add a suffix to the version?
```suggestion
            applicationIdSuffix = ".debug"
            versionNameSuffix "-debug"
```